src/Components/App/App.tsx

```diff
@@ -1,22 +1,81 @@
-import './App.css';
-import NavBar from '../NavBar/NavBar';
-import { Route, Routes } from 'react-router-dom';
-import Homepage from '../Homepage/Homepage';
+import "./App.css";
+import React from "react";
+import { useEffect, useState } from "react";
+import { Route, Routes } from "react-router-dom";
+import Homepage from "../Homepage/Homepage";
+import { generateID, cleanUpData, ArtData} from "../../utils";
 import SavedPage from '../SavedPage/SavedPage';
+import NavBar from '../NavBar/NavBar';
+
+function App() {
+  const [pieces, setPieces] = useState<ArtData[]>([]);
+  const [savePieces, setSavePieces] = useState<ArtData[]>([]);
+
+  const getArt = (objectID: number) => {
+    console.log(" obj ID", objectID);
+    return fetch(
+      `https://collectionapi.metmuseum.org/public/collection/v1/objects/${objectID}`
+    ).then((res) => res.json());
```

Comment:
Eventually you'll want to add error handling to check for an ok response here, then add a catch block to actually handle any errors after the fetch is invoked.